### PR TITLE
CS: remove health reqs for omega and curly

### PR DESCRIPTION
**Pull request by Miepee**
Darn energy requirements strike again

### CODE REVIEW

File: randovania/games/cave_story/logic_database/Sand Zone.txt [hunk 1]
```diff
@@ -203,10 +203,7 @@ Extra - starting_script: <TRA0010:0099:0036:0033
                       Any of the following:
                           Missiles ≥ 11 and Kill Bosses with Missiles (Beginner)
                           Missiles ≥ 10 and Kill Bosses with Missiles (Intermediate)
-              Any of the following:
-                  # Energy Requirements
-                  Combat (Intermediate) or Normal Damage ≥ 19
-                  Combat (Beginner) and Normal Damage ≥ 10
+              Trivial
```
Comment (duncathan) [inline randovania/games/cave_story/logic_database/Sand Zone.txt:206]: toss a `FIXME: energy requirements here cause generation failures` comment on

File: randovania/games/cave_story/logic_database/Sand Zone.txt [hunk 2]
```diff
@@ -379,15 +376,10 @@ Extra - starting_script: <TRA0029:0090:0018:0009
...
-      All of the following:
-          Any of the following:
-              # Item requirements
-              Can Kill Bosses
-              Missiles ≥ 9 and Super Missile Launcher and Kill Bosses with Missiles (Beginner)
-          Any of the following:
-              # Health requirements
-              Combat (Intermediate) or Normal Damage ≥ 14
-              Combat (Beginner) and Normal Damage ≥ 9
+      Any of the following:
```
Comment (duncathan) [inline randovania/games/cave_story/logic_database/Sand Zone.txt:379]: ditto here

**Review by duncathan — CHANGES_REQUESTED** [review]

File: randovania/games/cave_story/logic_database/Sand Zone.txt (hunk 1 above)
Comment (Miepee) [reply randovania/games/cave_story/logic_database/Sand Zone.txt:206]: i have a TODO, I'll change it to a fixme

the pretty-print that generates the txt files doesnt show it, due to it being trivial :(

Comment (duncathan) [reply randovania/games/cave_story/logic_database/Sand Zone.txt:206]: in that case make it a 1 damage req for now. idc about the edge case of presets starting with 1hp

Comment (Miepee) [reply randovania/games/cave_story/logic_database/Sand Zone.txt]: for some reason there's now 1 resolver failure :(

Comment (Miepee) [reply randovania/games/cave_story/logic_database/Sand Zone.txt]: Nevermind, it seems i didnt look through things correctly, and this is caused by something else.

So this PR bumps generation from 96/100 to 99/100.

Currently trying to figure out what the last failure is caused by

Comment (Miepee) [reply randovania/games/cave_story/logic_database/Sand Zone.txt]: Found it, it was called toroko+.

**Review by duncathan — APPROVED** [review]

**Pull request merged**